tests/zero_code_change/utils.py

```diff
@@ -54,3 +54,10 @@ def build_json(
     with open(path.absolute(), "w") as outfile:
         json.dump(config_dict, outfile)
     return path.absolute()
+
+
+def launch_job(script_file_path, script_args, config_file_path):
```

Comment:
where is this used?

Reply:
This is not used anywhere.
I have squashed and rebased the branch.
Apologies for the wrong diff.